CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.25)
```

Comment:
I would suggest not using the nearly latest version of CMake, but allowing slightly older versions, especially as newer features aren't used in this build script.

Reply:
For now the CMake version is set to the lowest required level to produce a build.  In order to sustain the intent of cmake_minimum_required.  It certainly can be used with a more recent version but why restrict?  The latest versions will still work with 3.25.

Reply:
Which features are you using that specifically require CMake 3.25?

Reply:
The main use case is ensuring the find_* set of functions correctly handle frameworks.  While 3.25 doesn't support xcframeworks correctly, none of the dependences are using xcframeworks thus don't need to bump it higher.

If prior versions can support frameworks without issue then it's fine to bring it back down; but in my local testing, 3.25 was the first one that works end to end.

Reply:
Hmm, that's weird. The [find_library docs](https://cmake.org/cmake/help/latest/command/find_library.html) don't mention anything about finding frameworks being a feature added in a version past CMake 3.0, which makes me wonder if what you were experiencing was due to some other issue?

Reply:
You're welcome to discover any issue you can find with the PR, infact that'd be very helpful.  If you find that a prior version of CMake works, then I'm happy to accept any changes you suggest.
While I don't pretend to have direct knowledge over any issues in find_library discovering frameworks: https://gitlab.kitware.com/cmake/cmake/-/issues/21752 that one is why I'm on an even more recent CMake then 3.25.  While that is specific to xcframeworks, when I was on 3.25 everything here works.